.github/workflows/pythonpackage.yml

```diff
@@ -31,17 +31,16 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install dependencies coverage, codecov, flake8, fastimport, mypy
```

Comment:
Why list the individual dependencies, especially if you're moving them out of this file?

Reply:
I just want to explain which kits are installed, I will change it back

Reply:
That doesn't seem unreasonable, but you're moving the actual list to somewhere else so now there are two places to keep up to date if the development dependencies change.

Reply:
Yes, there is nothing wrong, it is currently the case. Although I am not sure if I understand your thoughts.